ansible/roles/operator-pipeline/tasks/pipeline-secrets.yml

```diff
@@ -272,8 +272,9 @@
           suffix: "{{ suffix }}"
           env: "{{ env }}"
       data:
-        username: "{{ lookup('file', operator_pipeline_hydra_username_local_path, rstrip=False) | b64encode }}"
-        password: "{{ lookup('file', operator_pipeline_hydra_password_local_path, rstrip=False) | b64encode }}"
+        sso_token_url: "{{ operator_pipeline_hydra_sso_token_url | b64encode }}"
```

Comment:
The SSO token url is just plain text in the config and not really that private. I don't think it's necessary to save it inside a secret just to extract it back out later?